src/ansible_creator/arg_parser.py

```diff
@@ -305,9 +287,10 @@ def _add_resource_role(self, subparser: SubParser[ArgumentParser]) -> None:
             "path",
             default="./",
             metavar="path",
-            help="The path to the Ansible collection. The default is the "
-            "current working directory.",
+            help="The path to the role directory. The default is the current working directory.",
```

Comment:
We should make this consistent with the plugins, let the use provide the path to the root of an ansible collection.  The goal here is to scaffold a new role into a collectiion, not a stand alone role.